install/assets/install.sql

```diff
@@ -366,92 +366,431 @@ INSERT INTO `options` (`option_id`, `option_name`, `option_value`, `autoload`) V
-INSERT INTO `timezones` VALUES ('88', '-5.0', '(GMT-05:00)-Eastern Time (US & Canada)');
-INSERT INTO `timezones` VALUES ('90', '-4.0', '(GMT-04:00)-Atlantic Time (Canada)');
-INSERT INTO `timezones` VALUES ('91', '-4.0', '(GMT-04:00)-Caracas, La Paz');
-INSERT INTO `timezones` VALUES ('92', '-4.0', '(GMT-04:00)-Santiago');
-INSERT INTO `timezones` VALUES ('94', '-3.0', '(GMT-03:00)-Brasilia');
-INSERT INTO `timezones` VALUES ('95', '-3.0', '(GMT-03:00)-Buenos Aires, Georgetown');
-INSERT INTO `timezones` VALUES ('98', '-1.0', '(GMT-01:00)-Azores');
-INSERT INTO `timezones` VALUES ('99', '-1.0', '(GMT-01:00)-Cape Verde Is.');
-INSERT INTO `timezones` VALUES ('100', '0.0', '(GMT)-Casablanca, Monrovia');
-INSERT INTO `timezones` VALUES ('102', '1.0', '(GMT+01:00)-Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna');
-INSERT INTO `timezones` VALUES ('103', '1.0', '(GMT+01:00)-Belgrade, Bratislava, Budapest, Ljubljana, Prague');
-INSERT INTO `timezones` VALUES ('106', '1.0', '(GMT+01:00)-West Central Africa');
-INSERT INTO `timezones` VALUES ('107', '2.0', '(GMT+02:00)-Athens, Beirut, Istanbul, Minsk');
-INSERT INTO `timezones` VALUES ('108', '2.0', '(GMT+02:00)-Bucharest');
-INSERT INTO `timezones` VALUES ('111', '2.0', '(GMT+02:00)-Helsinki, Kyiv, Riga, Sofia, Tallinn, Vilnius');
-INSERT INTO `timezones` VALUES ('114', '3.0', '(GMT+03:00)-Kuwait, Riyadh');
-INSERT INTO `timezones` VALUES ('116', '3.0', '(GMT+03:00)-Nairobi');
-INSERT INTO `timezones` VALUES ('121', '5.0', '(GMT+05:00)-Ekaterinburg');
-INSERT INTO `timezones` VALUES ('122', '5.0', '(GMT+05:00)-Islamabad, Karachi, Tashkent');
-INSERT INTO `timezones` VALUES ('123', '5.5', '(GMT+05:30)-Chennai, Kolkata, Mumbai, New Delhi');
-INSERT INTO `timezones` VALUES ('124', '5.8', '(GMT+05:45)-Kathmandu');
-INSERT INTO `timezones` VALUES ('126', '6.0', '(GMT+06:00)-Astana, Dhaka');
-INSERT INTO `timezones` VALUES ('127', '6.0', '(GMT+06:00)-Sri Jayawardenepura');
-INSERT INTO `timezones` VALUES ('129', '7.0', '(GMT+07:00)-Bangkok, Hanoi, Jakarta');
-INSERT INTO `timezones` VALUES ('131', '8.0', '(GMT+08:00)-Beijing, Chongqing, Hong Kong, Urumqi');
-INSERT INTO `timezones` VALUES ('135', '8.0', '(GMT+08:00)-Taipei');
-INSERT INTO `timezones` VALUES ('136', '9.0', '(GMT+09:00)-Osaka, Sapporo, Tokyo');
-INSERT INTO `timezones` VALUES ('139', '9.5', '(GMT+09:30)-Adelaide');
-INSERT INTO `timezones` VALUES ('141', '10.0', '(GMT+10:00)-Brisbane');
-INSERT INTO `timezones` VALUES ('142', '10.0', '(GMT+10:00)-Canberra, Melbourne, Sydney');
-INSERT INTO `timezones` VALUES ('144', '10.0', '(GMT+10:00)-Hobart');
-INSERT INTO `timezones` VALUES ('146', '11.0', '(GMT+11:00)-Magadan, Solomon Is., New Caledonia');
-INSERT INTO `timezones` VALUES ('147', '12.0', '(GMT+12:00)-Auckland, Wellington');
-INSERT INTO `timezones` VALUES ('149', '13.0', '(GMT+13:00)-Nuku\'alofa ');
-INSERT INTO `timezones` VALUES ('150', '-4.5', '(GMT-04:30)-Caracas');
+INSERT INTO `timezones` (`id`, `name`) VALUES
```

Comment:
One of the biggest problems of this PR.

You changed the install.sql in the installer. This is not how database changes work. It's the same with your additional column of the utc option in the users table. 

You did the change to your database. So it works for you. But this would require that all users which use Wavelog would need to change their database manually. Otherwise their installation would be broken. 

What you would need to do is a migration which modifies all databased. Changing the install.sql is always a bad idea and should be really good considered..